errors/src/androidMain/kotlin/dev/icerock/moko/errors/presenters/SelectorErrorPresenter.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.errors.presenters
+
+import androidx.fragment.app.FragmentActivity
+
+actual class SelectorErrorPresenter<T : Any> actual constructor(
```

Comment:
@Tetraquark review it please :)

Reply:
I think the mapper no need in ErrorPresenters anymore because all data comes to the `show` method. I pushed that changes where `ErrorPresenter` again becomes as interface :)

Reply:
yeah! it's cool!